sql/core/benchmarks/ExternalAppendOnlyUnsafeRowArrayBenchmark-results.txt

```diff
@@ -42,4 +42,8 @@ Spilling with 10000 rows:                 Best Time(ms)   Avg Time(ms)   Stdev(m
 UnsafeExternalSorter                                 11             11           1         14.7          68.0       1.0X
 ExternalAppendOnlyUnsafeRowArray                      9             10           1         17.1          58.5       1.2X
 
-
+OpenJDK 64-Bit Server VM 1.8.0_252-8u252-b09-1~16.04-b09 on Linux 4.4.0-178-generic
+Intel(R) Xeon(R) CPU E5-2687W v3 @ 3.10GHz
+Spilling  SpillReader with 16000 rows:    Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------------------------------
+UnsafeSorterSpillReader_bufferSize1024              411            426          13          0.6        1607.2       1.0X
```

Comment:
What's a number without this PR?

Reply:
Here it is:
OpenJDK 64-Bit Server VM 1.8.0_252-8u252-b09-1~16.04-b09 on Linux 4.4.0-178-generic
Intel(R) Xeon(R) CPU E5-2687W v3 @ 3.10GHz
Spilling  SpillReader with 16000 rows:    Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
//------------------------------------------------------------------------------------------------------------------------
UnsafeSorterSpillReader                          932683         943898         NaN          0.0     3643292.4       1.0X

Reply:
There are three spill files. It will result in three I/Os for each created iterator, because it has to read number of rows from each spilled file. So, we see more than 2000 X performance difference.

Reply:
oh, looks nice.